llvm/test/Assembler/invalid-c-style-comment1.ll

```diff
@@ -0,0 +1,10 @@
+; RUN: not llvm-as --disable-output %s 2>&1 | FileCheck %s -DFILE=%s
+
+@B = external global i32
+
+*/
```

Comment:
I'm referring to the `*/` here, not to the comment below. I'd expect a closing `*/` without opening `/*` to generate an error.

Reply:
You're right, I think you found a bug in the code :) I'll remove tis stray */ here and add another unit test for this.